blog/20250205-virtual-hackathon/index.md

```diff
@@ -0,0 +1,114 @@
+* Yao-Ting Yao/ YaoTingYao/ Clark CGA/Mint Tea
+* Martin Renou / martinRenou / QuantStack / Coffee
+* Greg Mooney / gjmooney / QuantStack / Pumpkin spice latte if I'm feeling fancy
+* Arjun Verma / arjxn-py / QuantStack / Cold Coffee
+* Brookie Guzder-Williams/ brookisme / Coffee
+* Qiusheng Wu / giswqs / University of Tennessee / Tea
+
+
+
+## Agenda & notes
+
+### ⚡ (5 minutes) Lightning intros
+
+Tell us about you in 30 seconds or less!
+
+
+### 💡 (5 minutes) Idea / team forming
+
+* What do you want to work on today?
+* Add your ideas to the “Hack teams” section below.
+* Add your favorite emoji next to ideas you’re excited about. Press the colon (:) key on your keyboard or navigate to “Insert > Emoji” in the menu bar to open the emoji browser.
+* Form teams from the ideas generated in the steps above! Add your objectives to the “Hack teams” section of the doc below.
+* Create breakout rooms.
+
+
+### 🪄 (all the minutes) Hack together!
+
+### 💬 (10 minutes) Share out
+
+* Save your progress in GitHub or Zulip as appropriate!
+  Please write for people who don’t have full context; link to related issues and documentation!
+* Fill in the “Share out” section of the doc below.
+
+
+## Hack teams
+
+For ideas, check out the [hackathon](https://github.com/geojupyter/jupytergis/labels/hackathon) and [good first issue](https://github.com/geojupyter/jupytergis/labels/good%20first%20issue) labels on the JupyterGIS project!
+
+Join the corresponding breakout room to hack!
+
+**Lobby**: Orientation! Help setting up dev environment, etc. Docs: https://jupytergis.readthedocs.io/en/latest/contributor_guide/index.html
+* Kristin
+
+1. Dockerize! (https://github.com/geojupyter/jupytergis/issues/267)
+    * Matt & Sanjay
+    * https://github.com/geojupyter/jupytergis-docker
+    * Martin: Could a GHA bot run every day and check for a new release? Should it work like the conda forge bot where it opens a PR, we review, and merge. Or just have the bot commit and tag? (the latter seems preferred)
+        * Arjun: Put dockerfile in JGIS repository in `scripts/` directory?
+        *
+2. Python API improvements (https://github.com/geojupyter/jupytergis/issues/436)
+    * Brookie & Martin & Arjun & Yao-Ting
+    * Walked through workflows.
+      Brookie will clean up notes and put on GitHub as a comment to the issue: https://github.com/geojupyter/jupytergis/issues/436
+      Also start a chat on Zulip!
+
+#### More
+
+3. Layer show/hide button gets pushed off the panel (https://github.com/geojupyter/jupytergis/issues/244)
+4. Improve layer gallery contributor experience (https://github.com/geojupyter/jupytergis/issues/277)
+    * Martin: The gallery is generated from xyzservices Python package. How's the contributor experience for this package?
+    * https://github.com/geojupyter/jupytergis/blob/b5cdceb81d316efe9dd520671a626cc533108e7f/packages/base/rasterlayer_gallery_generator.py#L9
+
+
+## Share out
+
+Think about:
```

Comment:
Feel free to add about your docker image thingy :)

Reply:
[7b3d476](https://github.com/geojupyter/geojupyter.org/pull/32/commits/7b3d4760aa8f5c058fa49b44366d72da27783611)

Good call :)